### DISCUSSION TITLE
Questions related to low-level output

### DISCUSSION THREAD

**Discussion by liyuchen88427** [yalmip/YALMIP#948] [Problems using YALMIP] [answered]
I am trying to program the Gomory Cutting plane algorithm which requires a Simplex Tableau. I am using the Gurobi solver and syntax ('savesolveroutput',1) to extract it. However, the output does not include the information regarding the tableau. Could you guys advise on that. I appreciate your help very much!

Steven

**Comment by johanlofberg** [comment, marked as answer]
If it is not available in the output after turning on savesolveroutput, there is no way to access it via YALMIP as Gurobi isn't returning the data

**Reply by liyuchen88427** [reply]
It is so unfortunate.  I try the matlab built-in function linprog() and it does not provide the final tableau as well. 
Johan, do you have another solution? Many thanks.

**Reply by johanlofberg** [reply]
Not besides exporting all the data of the model, the primal and the dual, and then manually creating it somwhow

**Reply by liyuchen88427** [reply]
I wil try my best to find a document, but I doubt it actually exisits.
